package com.github.aivanovski.picoautomator.cli.entity.exception

class TestExecutionException : PicoAutomatorException(message = "Test fail")